exts/cesium.omniverse/cesium/omniverse/ui/credits_parser.py

```diff
@@ -2,60 +2,69 @@
 import omni.kit.pipapi
 import omni.ui as ui
 import webbrowser
-from typing import Optional, List, Tuple
+from typing import Optional, List, Tuple, NamedTuple
 from .uri_image import CesiumUriImage
 from .image_button import CesiumImageButton
 
 _num_retries = 0
 
 
+class ParsedCredit(NamedTuple):
```

Comment:
So, inheriting `NamedTuple` technically does work here, but this would be far better as a `@dataclass`. While data classes are mutable, they tend to be faster and have real support for default values.

https://peps.python.org/pep-0557/#why-not-just-use-namedtuple